Makefile

```diff
@@ -50,6 +49,10 @@ $(BIN)/gometalinter: REPOSITORY=github.com/alecthomas/gometalinter
 
 # Useful commands
 
+.PHONEY: test
+test: $(BASE); $(info $(M) running test…)
+	$Q cd $(BASE) && $(GO) test -v ./... -cover
```

Comment:
we use this often: - ginkgo -r --randomizeAllSpecs --randomizeSuites --failOnPending --cover --trace --race --progress